keras_nlp/samplers/top_p_sampler_test.py

```diff
@@ -88,6 +88,28 @@ def test_early_stopping(self):
         )
         self.assertEqual(self.join_as_string(output), ["sequentzzzzz"])
 
+    def test_sample_from_all_tokens(self):
```

Comment:
I don't think this test adds to much. If we want to add a test here, it should test that our cutoff point is working.

e.g. `p=1.0`, `k=5`, uniform logits, assert that all outputs in the first five vocab options.

Reply:
This test case is just a sanity check on sampler call with `k=None` works.

Agree the proposed test works better! changed.